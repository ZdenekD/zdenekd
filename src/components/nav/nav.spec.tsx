import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '../../__test__/utils/helpers';
import Nav from '.';

describe('Navigation', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<Nav />);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-nav');

        expect(component.exists()).toBe(true);
    });
});
