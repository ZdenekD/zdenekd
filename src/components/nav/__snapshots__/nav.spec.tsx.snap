// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation match snapshot 1`] = `
<nav class="nav"
     data-test="component-nav"
>
  <ul class="list">
    <li>
      <a href="/"
         class="anchor "
         data-test="component-anchor"
      >
        Úvodní stránka
      </a>
    </li>
    <li>
      <a href="/projects"
         class="anchor "
         data-test="component-anchor"
      >
        Projekty
      </a>
    </li>
    <li>
      <a href="/contact"
         class="anchor "
         data-test="component-anchor"
      >
        Kontakt
      </a>
    </li>
  </ul>
</nav>
`;
